### ISSUE TITLE
[SF][A11y][MessageView]: [initial focus is not set when expand the detail view]

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents-react#7059]
### Describe the bug

1. When the expandable info section is activated, the initial focus is not set. The user must press the Tab key to locate the focus indicator. And on returning back from the expanded section, focus is lost. 

2. No intimation on the presence of expandable section and neither activating keys are announced.

Please see the storybook: https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/user-feedback-messageview--docs

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/user-feedback-messageview--docs

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

1. Ensure focus is not lost and is visible when the section is expanded. 
2. Ensure to provide screen reader announcements when sections are expanded. Also make sure the activation keys should be announced.

### UI5 Web Components for React Version

2.8.0

### UI5 Web Components Version

2.8.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @tandan0813 

please extend your description with clear steps to reproduce the issue. Otherwise we're unfortunately not able to assist you.

**Comment by Lukas742** [comment]
Hi @tandan0813 

thanks for the clarification. [This PR](https://github.com/SAP/ui5-webcomponents-react/pull/7098) will fix the first issue and also add an announcement ("Has Details") if details are available. 
I didn't find specifications for a special announcement regarding the keys, so for this to be implemented this request would need to go through the Inner Source process and be synchronised with the SAP Central Design team first. In case this is a requirement for you, please open another issue for this and include the ticket number of the central design ticket there for reference.

**Comment by tandan0813** [comment]
Hi @Lukas742  Thanks for your help!!
